sentry/sentry.go

```diff
@@ -20,12 +22,13 @@ var (
 type Options sentry.ClientOptions
 
 type Hook struct {
-	client      *sentry.Client
-	levels      []logrus.Level
-	tags        map[string]string
-	release     string
-	environment string
-	prefix      string
+	client       *sentry.Client
+	levels       []logrus.Level
+	tags         map[string]string
+	release      string
+	environment  string
+	prefix       string
+	flushTimeout time.Duration
```

Comment:
Need a default value.

Reply:
Added 10sec timeout as default value in NewHook method.